packages/bpk-component-segmented-control/README.md

```diff
@@ -0,0 +1,24 @@
+# bpk-segmented-control
+
+## Installation
+Check the main [Readme](https://github.com/skyscanner/backpack#usage) for a complete installation guide.
+
+## Usage
+```js
+import BpkSegmentedControl from '@skyscanner/backpack-web/bpk-component-segmented-control';
+
+export default () => (
+  <BpkSegmentedControl
+  buttonContents={buttonContent}
+  onItemClick={() => {}}
+  selectedIndex={1} // button selected on load
+  type={SEGMENT_TYPES.SurfaceContrast}
+  shadow
+  />
+)
+```
+
+
+## Props
+// doc link to be added
```

Comment:
I'm not sure how this link gets generated in your docs? Could you clarify please?

Reply:
We just link to our docs site here as storybook handles this:

So you would do this, example https://github.com/Skyscanner/backpack/blob/main/packages/bpk-component-autosuggest/README.md

```md

## Props

Check out the full list of props on Skyscanner's [design system documentation website](https://www.skyscanner.design/latest/components/autosuggest/web-1dnVX8RX#section-props-01).
```